app/schemas/project_create_schema.rb

```diff
@@ -19,6 +19,10 @@ class ProjectCreateSchema < JsonSchema
       description "Two character ISO 638 language code, optionally include two character ISO 3166-1 alpha-2 country code seperated by a hyphen for specific locale. ie 'en', 'zh-tw', 'es_MX'"
     end
 
+    property "private" do
+      type "boolean"
```

Comment:
Can we make this field required in the schema if we're going to require it to create a project? See https://github.com/camallen/Panoptes/blob/projects_private_fix/app/schemas/subject_set_create_schema.rb#L5